lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "packages": ["packages/*", "vnext"],
+  "packages": ["packages/**", "vnext"],
```

Comment:
Not sure how much is really matters, (how lerna does its file trolling?), but maybe packages/* and packages/@rnw-scripts/* would save it some effort?

Reply:
Running glob.sync for packages/** as a test of the worst case was really slow. I'll make that change.